A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.html

```diff
@@ -81,4 +81,5 @@
 <script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A32NX_LocalVarUpdater.js"></script>
 <script type="text/html" import-script="/Pages/A32NX_Core/A32NX_APU.js"></script>
 <script type="text/html" import-script="/Pages/A32NX_Core/A32NX_Core.js"></script>
+<script type="text/html" import-script="/Pages/A32NX_Core/A32NX_BrakeTemp.js"></script>
```

Comment:
shouldn't this go above the line that imports A32NX_Core.js?

Reply:
maybe it still works, but i think it would look nicer above it, to better show that it gets called from the core and not the other way around